aptos-move/move-examples/drand/sources/drand.move

```diff
@@ -0,0 +1,136 @@
+/// The drand randomness beacon outputs a piece of randomness `r_i` for every round `i` such that anybody can verify
```

Comment:
should we add this drand contract to the framework instead?

Reply:
I think it would be a bit premature: there's many other randomness beacons we (could) support and it's unclear what the criteria should be for adding one to the framework. An example works fine, I think.